helm-chart-sources/pulsar/values.yaml

```diff
@@ -327,9 +327,9 @@ image:
     pullPolicy: IfNotPresent
     tag: 1.0.0
   burnell:
-    repository: datastax/burnell
+    repository: meticulo3366/burnell
     pullPolicy: Always
-    tag: 1.0.0
+    tag: rootless
```

Comment:
We'll want to release an official version of the non root datastax burnell image and then use that version here.

Reply:
let me try building